docs/BuildErrors.md

```diff
@@ -80,6 +80,10 @@ When attempting to restore servicing tags e.g. `v3.1.7`,  the NuGet.config file
 
 The `darc-int-...` feeds in NuGet.config are used only when building internally and are not needed after the tags are created. Delete all such entries in the file and retry.
 
+## Error: Generated code is not up to date in eng/ProjectReferences.props.
+
+After adding or moving project you need to update the two `DotNetProjects Include` lists in `eng/Build.props`
```

Comment:
This requirement is almost never hit. I'm not sure the change is worthwhile. If you feel strongly about adding something, alright but it'll need to be toned down e.g.
```suggestion
After some project additions or moves, you may need to update the two `DotNetProjects Include` lists in `eng/Build.props`
```

Reply:
Sounds better, thanks!